src/server/server.ts

```diff
@@ -324,6 +360,38 @@ function compileCommon(req: express.Request, res: express.Response, wrapWithRunn
         });
 }
 
+const parseXmlString = (xml) => {
+    return new Promise((resolve, reject) => {
+        xml2js.parseString(xml, (err, result) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(result);
+            }
+        });
+    });
+};
+
+function getVersionNumber(): Promise<any> {
+     return new Promise((resolve, reject) => {
+        return Request.post({
+            url: 'https://store.office.com/app/query?type=5&cmo=en-US&rt=xml',
+            body: 'wa104380862', /* Script Lab ID */
+        }, (error, httpResponse, body) => {
+            if (error) {
+                ai.trackEvent('[Snippet] Open in playground failed', { error });
```

Comment:
The telemetry should be more specific if logged in this function such as Get Version number failed.